apps/astarte_pairing/lib/astarte_pairing_web/controllers/fdo_onboarding_controller.ex

```diff
@@ -55,4 +55,31 @@ defmodule Astarte.PairingWeb.FDOOnboardingController do
         |> send_resp(400, err)
     end
   end
+
+  def prove_device(conn, _params) do
+    realm_name = conn.params["realm_name"]
+    cbor_body = conn.assigns[:cbor_body]
+    session_key = conn.assigns.to2_session.key
+    device_id = conn.assigns.to2_session.device_id
+    public_key = conn.assigns.to2_session.secret
+    nonce = conn.assigns.to2_session.nonce
+
+    with {:ok, response} <-
+           OwnerOnboarding.prove_device(
+             realm_name,
+             cbor_body,
+             public_key,
+             device_id,
+             session_key,
+             nonce
+           ) do
```

Comment:
let's have the session be a known structure within our logic, no need to always use base values

```suggestion
           OwnerOnboarding.prove_device(
             realm_name,
             cbor_body,
             public_key,
             conn.assigns.to2_session
           ) do
```